src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -14,12 +14,43 @@
 * limitations under the License.
 *******************************************************************************/
 
+#include <CPP/eltwise.hpp>
+#include <CPP/input_layout.hpp>
+#include <CPP/layout.hpp>
+#include <CPP/network.hpp>
+#include <CPP/scale.hpp>
+
 #include "ngraph/runtime/intelgpu/intelgpu_backend.hpp"
+#include "ngraph/runtime/intelgpu/intelgpu_layout.hpp"
 #include "ngraph/runtime/intelgpu/intelgpu_tensor_view.hpp"
 
 using namespace std;
 using namespace ngraph;
 
+void do_eltwise_operation(cldnn::topology& topology,
+                          const shared_ptr<Node>& op,
+                          cldnn::eltwise_mode mode)
+{
+    if (op->get_output_size() != 1 || op->get_input_size() != 2)
+    {
+        ostringstream os;
+        os << "eltwise operation \"" << op->description() << "\" input and output sizes mismatch.";
```

Comment:
It is nice to know what the sizes where.
Spell out the word `Elementwise` as this message is intended for someone who is not you.